run_test.py

```diff
@@ -15,7 +15,7 @@ class TestHash(unittest.TestCase):
     def test(self):
         self.maxDiff = None
         exceptions = []
-        for fname in glob.glob("images/*.png"):
+        for fname in glob.glob("images/v4/*.png"):
             phash = imagehash.phash(Image.open(fname), hash_size=_HASH_SIZE)
```

Comment:
Do we have the ability to pass the phash version to the function, or does imagehash not maintain backwards compatible image hashing functionality?

Reply:
It's a breaking change moving to v4. There's no way to generate the old hash from v4, hence the test coverage is only on the new images using the new imagehash v4.

The test the old hashes we'd need to install the pre-v4 imagehash ... and I'm kinda loathed to do that. Is that a safe approach to take?